src/common/components/game/VolumeControl.tsx

```diff
@@ -0,0 +1,108 @@
+import * as React from 'react';
+import IconButton from '@material-ui/core/IconButton';
+import Icon from '@material-ui/core/Icon';
+import { withStyles, WithStyles } from '@material-ui/core/styles';
+import Slider from '@material-ui/lab/Slider';
+
+import { MAX_Z_INDEX } from '../../constants';
+
+import Tooltip from '../Tooltip.js';
+
+const styles = {
```

Comment:
I would put this as a static inside `VolumeControl`, like I did with `SetSummary`.